libevmasm/ConstantOptimiser.h

```diff
@@ -141,6 +146,8 @@ class ComputeMethod: public ConstantOptimisationMethod
 protected:
 	/// Tries to recursively find a way to compute @a _value.
 	AssemblyItems findRepresentation(u256 const& _value);
+	/// Recomputes the value from the representation and checks for correctness.
+	bool checkRepresentation(u256 const&);
```

Comment:
All the other functions here are stateful, too, so I think it is fine to use the member, especially for a protected function. Will add the parameter name, though.